gwtlib/gc.py

```
@@ -0,0 +1,459 @@
+    except Exception as e:
+        print(f"  Error running clean command: {e}", file=sys.stderr)
+        return False
+    finally:
+        os.chdir(current_dir)
+
+
+def execute_gc_plan(
+    plan: GcPlan,
+    git_dir: str,
+    clean_cmd: Optional[str] = None,
+    dry_run: bool = False,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)